### DISCUSSION TITLE
Trigger a point in xychart from outside

### DISCUSSION THREAD

**Discussion by thomaslein** [airbnb/visx#1680] [General]
Hi, thanks for a awesome library. 
I have this usecase and I'm wondering if it's possible.
Is it possible to set a datapoint/index in a xychart from outside the chart?
The usecase is to mouseover one chart and set the same datapoint/index in two other charts so they sync markers on mouseover.

**Comment by williaster** [comment]
hey @thomaslein thanks for checking out visx. going to convert this to a discussion as it's a usage question rather than an issue. there is support for this advanced use-case, it'd documented in the [`xychart` docs](https://airbnb.io/visx/docs/xychart) here:

<img width="1086" alt="image" src="https://user-images.githubusercontent.com/4496521/227811280-08f34380-0220-4db1-973a-4ae59d9c6bc7.png">

Generally there are two approaches, programmatic triggering as you say (see linked example in docs) or you can actually have multiple charts share an `EventEmitter` context which is how tooltips are implemented in the chart/which is exposed for advanced use cases like this. there's another discussion with more examples if you want to pursue this https://github.com/airbnb/visx/discussions/1646

**Reply by thomaslein** [reply]
It works and it's awesome! Thanks for the help on this @williaster
